### PR TITLE
WIP - integrate storyengine with gdscript prototype

### PR DESCRIPTION
**Pull request by sabrinagannon**
This is my branch for integrating the csharp storyengine with gdscript, currently it gives this error when trying to build:
Screenshot 2022-11-06 at 10 15 54 PM
[Due to this warning in the godot docs](https://docs.godotengine.org/en/stable/tutorials/scripting/cross_language_scripting.html#instantiating-c-nodes-from-gdscript), I thought this was because the csharp class was different than the file name (classname was `StoryEngineApi` and filename was `StoryEngine.cs`) but when I tried changing this in the editor the error persisted which makes me think I'm not building it properly?
Are there special compile steps for the c# integration @gailcarmichael ? 

I also noticed I couldn't build the c# project itself, so maybe I do not have something installed; open to any and all ideas there!

### CODE REVIEW

File: gdscript_vn_prototype/dialogue_runner.gd [hunk 1]
```diff
@@ -1,4 +1,6 @@
 extends Control
+var storyengine = load("res://lib/StoryEngine/StoryEngine.cs")
+var storynode = storyengine.new("filename", "otherFilename")
```
Comment (sabrinagannon) [inline gdscript_vn_prototype/dialogue_runner.gd:3]: @gailcarmichael woof, sorry for all these whitespace changes and the lack of newline stuff, but these two lines are the changes I actually made (along with copying the lib folder into this project!)

File: TestScene/TestScene.cs [hunk 2]
```diff
@@ -3,15 +3,15 @@
 
 public class TestScene : Node2D
 {
-    // Declare member variables here. Examples:
-    // private int a = 2;
-    // private string b = "text";
```
Comment (gailcarmichael) [inline TestScene/TestScene.cs:8]: I noticed that spaces are being changed to tabs. We should probably try to stay consistent with that. Are you opposed to spaces to stick with what my code has used so far?

@alecMcgrail your thoughts welcome here too ^

File: gdscript_vn_prototype/dialogue_runner.gd (hunk 1 above)
Comment (gailcarmichael) [reply gdscript_vn_prototype/dialogue_runner.gd:3]: Oh I made the tab comment before seeing this haha.

Comment (gailcarmichael) [reply gdscript_vn_prototype/dialogue_runner.gd:3]: This is really cool if it's all it takes to use the C# stuff! I'll have to add some more to the API to test out even if it's just dummy stuff for now.

**Comment by sabrinagannon** [comment]
Gonna come back to this with fresh eyes!

**Pull request closed without merging**